src/Microsoft.VisualStudio.ProjectSystem.VisualBasic.VS/ProjectSystem/VS/Properties/OptionStrictEnumProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework.XamlTypes;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Properties
+{
+    [ExportDynamicEnumValuesProvider("OptionStrictEnumProvider")]
+    [AppliesTo(ProjectCapability.VisualBasic)]
+    internal class OptionStrictEnumProvider : IDynamicEnumValuesProvider
+    {
+        private readonly ICollection<IEnumValue> _listedOptionStrictEnumValues = new List<IEnumValue>
```

Comment:
@srivatsn The change that we discussed earlier to remove the `_listedOptionStrictEnumValues ` and use the values from the dictionary with the EnumValue's DisplayName set to 0 and 1 does not work. Before the Property page, specifically gets the value of the Name as the value instead of DisplayName. Check [Propertypage](http://index/?query=compileproppage2&rightProject=RoslynProjectSystem(Repo)&file=src%5CMicrosoft.VisualStudio.Editors%5CPropPages%5CCompilePropPage2.vb&line=516) and [EnumPropertyConverter.GetStandardValues](http://index/?leftProject=Microsoft.VisualStudio.ProjectSystem.VS.Implementation&leftSymbol=x5a24mknyyli&file=PropertyPages%5CDynamicType.cs)